test/multinomial.jl

```diff
@@ -98,3 +98,17 @@ r = fit_mle(Multinomial, x, fill(2.0, size(x,2)))
 @test r.n == nt
 @test length(r) == length(p)
 @test_approx_eq_eps probs(r) p 0.02
+
+# behavior for n = 0
+d0 = Multinomial(0, p)
+@test rand(d0) == [0, 0, 0]
+@test pdf(d0,[0, 0, 0]) == 1
+@test pdf(d0,[0, 1, 0]) == 0
+@test mean(d0) == [0, 0, 0]
+@test var(d0) == [0, 0, 0]
+@test cov(d0) == zeros(3, 3)
+@test entropy(d0) == 0
+@test insupport(d0,[0, 0, 0]) == true
+@test insupport(d0,[0, 0, 4]) == false
```

Comment:
Space after the comma here and on lines 111, 106, and 105 as well, please. :slightly_smiling_face: